subprojects/core-api/src/main/java/org/gradle/api/artifacts/MutableVersionConstraint.java

```diff
@@ -44,12 +44,27 @@
      */
     void setBranch(@Nullable String branch);
 
+    /**
+     * Sets the version as strict.
+     * <p>
+     * Any version not matched by this version notation will be excluded. This is the strongest version declaration.
+     * It will cause dependency resolution to fail if no version acceptable by this clause can be selected.
+     * This term supports dynamic versions.
+     * This clears any set rejected versions.
```

Comment:
I would move this sentence below the next one and make it a different paragraph as well.